pkg/v1/remote/transport/scope.go

```diff
@@ -22,3 +26,7 @@ const (
 	DeleteScope  string = PushScope
 	CatalogScope string = "catalog"
 )
+
+func DefaultScopes(ref name.Reference) []string {
```

Comment:
I find this more confusing than useful.  "Default" is too ambiguous a name.

Reply:
removed